cmd/root.go

```diff
@@ -192,6 +209,19 @@ func parseConfig(cmd *cobra.Command, conf *proxy.Config, args []string) error {
 	}
 	conf.DialerOpts = opts
 
+	// If a user has set telemetry-project, but hasn't enabled one of metrics or
+	// traces, fail.
+	if userHasSet("telemetry-project") &&
+		!userHasSet("telemetry-metrics") &&
+		!userHasSet("telemetry-traces") {
+		return newBadCommandError("cannot specify --telemetry-project without also setting --telemetry-metrics or --telemetry-traces")
```

Comment:
(nit:) Do we really need to split this line 3 ways? It's shorter than the next line, which we don't split. 

If we have to split it, I'd rather we use a variable over 2 lines rather than 3 imo